python/cugraph/traversal/traveling_salesman.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
```

Comment:
This and all others in this PR will need a 2021 date.

Reply:
Should all be updated now